src/Presentation/Presentation.csproj

```diff
@@ -4,5 +4,20 @@
         <TargetFramework>net5.0</TargetFramework>
         <Version>2021.0.0</Version>
     </PropertyGroup>
-    
+      <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.2" />
+
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="1.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer.Design" Version="1.1.0" PrivateAssets="All" />
+    <PackageReference Include="MySql.EntityFrameworkCore" Version="5.0.3.1" />
```

Comment:
Shouldn't all of those refereces be placed inside `Data` project? @danidoff Can you help us here, please?